### PR TITLE
fixed preview, show ghs-symbols and phrases

### PR DESCRIPTION
**Pull request by Epsilon02**

### CODE REVIEW

**Comment by Epsilon02** [comment]
MAK will be automatically shown as soon as #114 is merged

**Comment by crapStone** [comment]
image
could you add a linebreak in the header?

File: src/app/preview/preview.component.scss [hunk 1]
```diff
@@ -54,18 +54,26 @@ tr {
...
     }
   }
 }
 
+.padding-top-5px {
+  padding-top: 5px;
+}
+
+img {
+  width: auto;
```
Comment (6543) [inline src/app/preview/preview.component.scss:73]: I would not generalize for all imahes, instead infent a new class

**Review by crapStone — DISMISSED** [review]

File: src/app/preview/preview.component.scss (hunk 1 above)
Comment (Epsilon02) [reply src/app/preview/preview.component.scss:73]: This style is only in this component active and the ghs-symbols are the only images in this component.
I see no reason to create an extra class for that.

**Comment by 6543** [comment]
@Epsilon02 pleace resolve conflicts :D

File: src/app/preview/preview.component.ts [hunk 2]
```diff
@@ -57,7 +58,7 @@ export class PreviewComponent implements OnInit {
               name: value.name.modifiedData ?? value.name.originalData,
               cas: value.cas.modifiedData ?? value.cas.originalData,
               molecularFormula: value.molecularFormula.modifiedData ?? value.molecularFormula.originalData,
-              molarMass: value.molarMass.modifiedData ?? value.molecularFormula.originalData,
+              molarMass: value.molarMass.modifiedData ?? value.molarMass.originalData ?? '',
```
Comment (6543) [inline src/app/preview/preview.component.ts:61]:
```suggestion
              molarMass: value.molarMass.modifiedData ?? value.molarMass.originalData,
```
?

Comment (Epsilon02) [reply src/app/preview/preview.component.ts:61]: Doesn't work because originalData could also be undefined, this is a problem from our data model.

Comment (Epsilon02) [reply src/app/preview/preview.component.ts]: Well but you are right there was a better way.
I edited the related interface and now it works without this fallback.

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
with this the preview is finally complete :)

**Pull request merged**